README.md

```diff
@@ -39,6 +39,7 @@
 - ⚙️ Fully **customisable** behaviour [via "exchanges"](#-add-on-exchanges)
 - 🗂 Logical but simple default behaviour and document caching
 - ⚛️ Minimal React components and hooks
+- 🌱 Normalized caching via [`@urql/exchange-graphcache`](https://github.com/FormidableLabs/urql-exchange-graphcache)
```

Comment:
I've been wondering whether or not we want to have an "Ecosystem" section where we mention the normalized cache, the dev tools, `next-urql`, and `reason-urql`. I could also see those just becoming list items here, i.e.

```md
- ⚡️ SSR support via [`next-urql`](https://github.com/FormidableLabs/next-urql).
- DevTools via [`urql-devtools-exchange`](https://github.com/FormidableLabs/urql-devtools-exchange).
- Reason support via [`reason-urql`]https://github.com/FormidableLabs/reason-urql).
```

Reply:
Yea, we can probably change up this section: https://github.com/FormidableLabs/urql/blob/chore/improved-readme/README.md#-add-on-exchanges

Not sure what that'd look like, but shouldn't be too hard to figure out.